.bash_prompt

```diff
@@ -164,7 +164,14 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  # remote_branch="origin/$branch"
+  # Compute the remote tracking branch, if no remote branch then echo 0
+  remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2>/dev/null )"
```

Comment:
Sorry, I should have been more clear. I used 2 comments about the same line of code -- it was to show multiple steps of reasoning but wasn't clear which to use =/

Can you please update this line to include a fallback instead of exiting early? It's possible that user's don't have upstream branch tracking set up properly (happens a lot to my repos):

```bash
remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2> /dev/null || echo -n "origin/$branch")"
```

https://github.com/twolfson/sexy-bash-prompt/pull/69#discussion_r95015039